arangod/Cluster/FollowerInfo.cpp

```diff
@@ -542,3 +543,29 @@ VPackBuilder FollowerInfo::newShardEntry(VPackSlice oldValue) const {
   }
   return newValue;
 }
+
+uint64_t FollowerInfo::newFollowingTermId(ServerID const& s) noexcept {
+  uint64_t i = 0;
+  // We want the random number to be non-zero:
+  do {
+    i = RandomGenerator::interval(UINT64_MAX);
```

Comment:
one suggestion: should we compare `i` against the value of `_followingTermId[s]` here to prevent the RNG to create the same random number twice in a row?

Reply:
This is an interesting one. I have gone left and right on this one, without a clear preference. I will add this, and add a mutex to protect this map at the same time. Looking at this again it struck my eye that this is unprotected traffic.

Reply:
Done